data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -633,10 +633,15 @@
             [then]
                 [message]
                     race=dwarf
-                    message=_ "Lord Konrad, yon rumble was made by nae enemy o’ us. ’Tis the echoes of the very bedrock witherin’ and crumbling beneath us that we ’eard. Are ye feeling naught heat and loess redoubling around us? I beseech that we not tarry ’ere any more than we ha’ to, lest the forming abysses claim us."
+                    message=_ "Lord Konrad, yon rumble was made by nae enemy o’ us. ’Tis the echoes of the very bedrock witherin’ and crumbling beneath us that we ’eard. Are ye nae feeling the heat redoubling? Is yer breath nae difficult for the air, sowled by ashes? I beseech that we not tarry ’ere any more than we ha’ to, lest the forming abysses claim us."
```

Comment:
"difficult for the air" sounds like a strange phrasing. In fact... if anything it'd make more sense if "air" and "breath" were swapped, wouldn't it? "Is the air nae difficult for yer breath", that is. I dunno, if you have some reasoning behind your current phrasing that you can explain, maybe I could accept it, but...

Also "sowl" is a rare word so it might still be worth a po note.

Reply:
I think that swap does work. The only remaining strange thing is the use of "for", which... according to Wiktionary isn't _wrong_, but I feel like it's misleading to use that meaning of "for" in this context; "from" might work better.

Reply:
Yeah, I know which meaning of "for" youn intended (it's basically the "because" meaning). However, in the sentence "Is your breath not difficult for the air?", it sounds like you're saying the air is having a hard time because of your breath. Now, that's obviously nonsensical, so I guess people would figure it out. So maybe it's fine? I dunno, we should probably get another opinion on this.

Of the cited examples, I think only the last one is precisely the same meaning as you're going for here (try substituting "because" in the sentences; only two of them also require the addition of "of" to make sense). They don't have this strange ambiguity, either.

Reply:
Sorry, I completely overlooked that ambiguity. Thanks for explaining. Let's try either to change the sentence or to get another opinion on it. Could we change "difficult"? How about "toiled"?

Reply:
quibble, it should be: "nae enemy o' ours"

Other than that I like it a lot more now. This line definitely needs a PO comment explaining the archaic words

Reply:
Certainly, it should use the genitive, unless the character is speaking a dialect. I'll change it to "ours". Would "ours" be spelled with an apostrophe (like "o'" for "of") in dwarf speech?

"sowl" is already explained in a comment. I'll add po comments for some more words; let me know if I miss any. "Yon", "ye", and "nae" are pretty normal for dwarves (they're used in S15 in this campaign and all over THoT), so if they need an explanation it should go to the wiki, probably, where it can be made verbose.